packages/types/Calendar.d.ts

```diff
@@ -165,10 +165,22 @@ export interface CalendarEvent {
   seatsPerTimeSlot?: number | null;
 
   // It has responses to all the fields(system + user)
-  responses?: Prisma.JsonObject | null;
+  responses?: Record<
```

Comment:
Because CalendarEvent payload is common for Email, Calendar and Webhook, send it in this format. It would allow webhook payload to uniquely identify any field's response even if label changes. It also allows, Email and Calendars to show the current value of label instead of the name for the field